season_5/down_in_the_river_to_pray.ly

```diff
@@ -0,0 +1,729 @@
+
+  % page 3, system 1 (solo)
+  r4 b4 b4 cis
+  e4 e8 e8 e8 e8 e4
+  e1\fermata
+  fis8 fis8 fis8 gis fis4 e4
+  
+  % page 3, system 2 (solo)
+  gis4 fis4 e4\fermata cis4
+  b4 gis4 b4~ b8 cis8
+  e4 cis4 e4 gis4
+  fis2 e8 e8~ e8 cis8
+  b1
+  
+  % page 3, system 3 (duet)
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis8 e8~ e4
+  gis4 fis4 e4( cis8) dis8
+  e8 e8 cis4 b2
+  
+  % page 3, system 4 (duet)
+  \slashedGrace e16( fis2) fis4 e4 gis4 b4 fis8 e8~ e4
+  gis4 fis8 e8 e8 e8 cis4
+  e1~
+  e1
+
+  % page 4, system 1
+  r4 b4 b4 cis4
+  e4 gis8 a8 gis8 fis8 e4
+  fis4( e4 dis4 cis4)
+  dis8 e8 fis8 gis8 fis4 gis4
+
+  % page 4, system 2
+  a4 a4 gis4 gis4
+  gis4 e4 gis4~ gis8 a8
+  gis4 e4 gis8( a8) b8( cis8)
+  b2 a4 a8 a8
+
+  % page 5, system 1
+  gis1
+  gis2 gis4a4
+  b4 b4 a8( gis8~ gis4)
+  gis4 a4 b4 a8 a8
+
+  % page 5, system 2
+  gis8 a8 b4 a4( gis4)
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis16( e8.~ e4)
+  gis4 fis8 e8 e8 e8 cis4
+
+  % page 6, system 1
+  b2. r4
+  \time 4/4 \tempo 4=90
+  r1
+  r4 b4 b4 c4
+  e4 d8 d8 c8 c8 c4
+  
+  % page 6, system 2
+  b4( c4 b4 c4)
+  c8 d8 e8 e8 d4 c4
+  e4 e4 e4~ e8 c8
+  b4 g4 b4~ b8 c8
+
+  % page 7, system 1
+  e4 c4 e2(
+  fis4) e4 d4 c4
+  \time 3/2
+  b1~ b4 r4\fermata
+
+  % page 7, system 2
+  \time 2/2 \tempo 2=60
+  r4 b'4 b4 b4
+  b4 a4 a4( gis4)
+  gis4 fis4 gis4~ gis8 gis8
+
+  % page 8, system 1
+  a8 a8 gis4 a4( gis4)
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis16 gis8.~ gis4(
+  
+  % page 8, system 2
+  b4) b8 b8 b8 b8 b4
+  r2 a2
+  b1\fermata
+  
+}
+
+altoTwoMusic = \relative c' {
+  \key c \major
+  \numericTimeSignature
+  \time 2/2
+  \tempo 2=55
+  \set Score.markFormatter = #format-mark-circle-numbers
+
+  % lead-in
+  r1
+  r1
+
+  % alto/tenor duet
+  r1 r1 r1 r1 r1 r1 r1 r1 r1
+  r1 r1 r1 r1 r1 r1 r1 r1 r1
+  
+  % page 4, system 1
+  r4 b4 b4 cis4
+  e4 e8 e8 e8 e8 e8( dis8)
+  cis8( b8 cis2.)
+  dis8 e8 dis8 e8 dis4 e4
+
+  % page 4, system 2
+  e4 e4 dis4 e4
+  gis4 e4 gis4~ gis8 a8
+  gis4 e4 gis8( a8) b8( cis8)
+  b4( a4) gis4 fis8 fis8
+
+  % page 5, system 1
+  e1
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis8( e8~ e4)
+  gis4 fis4 e4 cis8 cis8
+
+  % page 5, system 2
+  e8 e8 cis4 b2
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis16( e8.~ e4)
+  gis4 fis8 e8 e8 e8 cis4
+
+  % page 6, system 1
+  b2. r4
+  \time 4/4 \tempo 4=90
+  r1
+  r4 b4 b4 c4
+  e4 d8 d8 c8 c8 c4
+  
+  % page 6, system 2
+  b4( c4 b4 a4)
+  b8 b8 c8 c8 b4 b4
+  b4 b4 b4~ b8 b8
+  b4 g4 b4~ b8 c8
+
+  % page 7, system 1
+  e4 c4 e2(
+  fis4) e4 d4 c4
+  \time 3/2
+  b1~ b4 r4\fermata
+
+  % page 7, system 2
+  \time 2/2 \tempo 2=60
+  r4 b'4 a8( gis8) fis4
+  gis4 gis4 fis4( gis4)
+  gis4 fis4 gis4~ gis8 gis8
+
+  % page 8, system 1
+  a8 a8 gis4 fis4( e4)
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis16 gis8.~ gis4(
+  
+  % page 8, system 2
+  b4) b8 b8 b8 b8 b4
+  r2 a2
+  gis1\fermata
+  
+}
+
+tenorOneMusic = \relative c''{
+  \key c \major
```

Comment:
The music has the Tenors using a tenor cleff (a G clef with an 8 under it).  In lilypond you can specify that with "\clef "treble_8"".  As a result, of this mistake, the tenor notes are all one octave too high.  That's easily corrected by deleting one of the apostrophe's from the relative starting pitch.